### DISCUSSION TITLE
CUDA error when trying to predict

### DISCUSSION THREAD

**Discussion by michelidiego** [autogluon/autogluon#4345] [Q&A]
Hi Everyone,

I'm using the TimeSeriesPredictor to fit a model using GPU. The fitting part goes well and nothing is wrong, but when I try to use the predict part (predictions = predictor.predict(train_data) I get an error related with CUDA:

RuntimeError: CUDA error: unknown error
CUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.
Compile with `TORCH_USE_CUDA_DSA` to enable device-side assertions.

From what I've read online the issue is probably with the code, but since autogluon comes packed , there's no way I've messed the code because if I disable the GPU and use only CPU, everything works fine.

Thank you for your help

**Comment by shchur** [comment]
Hi @michelidiego, can you please open a bug report under [issues](https://github.com/autogluon/autogluon/issues) and provide the relevant information such as full stack trace / minimal example to reproduce the issue?
